ui/src/features/components/ConfigurationForm/index.js

```diff
@@ -10,94 +10,286 @@ import {
-                    info: 'If true, don\'t fail requests on unverified server certificate errors',
-                    type: INPUT_TYPES.SWITCHER
+        this.Menu = [
+            {
+                name: 'internal_address',
+                key: 'internal_address',
+                floatingLabelText: 'Internal address',
+                info: 'The local ip address of your machine'
+            },
+            {
+                name: 'runner_docker_image',
+                key: 'runner_docker_image',
+                floatingLabelText: 'Docker image name',
+                info: 'The predator-runner docker image that will run the test'
+            },
+            {
+                name: 'runner_cpu',
+                key: 'runner_cpu',
+                floatingLabelText: 'Runner CPU',
+                info: 'The CPU allocated by each deployed runner',
+                valueType: 'float'
+            },
+            {
+                name: 'runner_memory',
+                key: 'runner_memory',
+                floatingLabelText: 'Runner memory (MB)',
+                info: 'Max memory to use by each deployed runner',
+                valueType: 'int'
+            },
+            {
+                name: 'minimum_wait_for_delayed_report_status_update_in_ms',
+                key: 'minimum_wait_for_delayed_report_status_update_in_ms',
+                floatingLabelText: 'Minimum delayed time for report update (ms)',
+                info: 'The minimum of time waiting for runner to report before the test is considered as finished',
+                valueType: 'int'
+            },
+            {
+                name: 'delay_runner_ms',
+                key: 'delay_runner_ms',
+                floatingLabelText: 'Delay runner (ms)',
+                info: 'Delay the predator runner from sending http requests (ms)',
+                valueType: 'int'
+            },
+            {
+                name: 'default_webhook_url',
+                key: 'default_webhook_url',
+                floatingLabelText: 'Default webhook url',
+                info: 'Default webhook url to send live report statistics to'
+            },
+            {
+                name: 'interval_cleanup_finished_containers_ms',
+                key: 'interval_cleanup_finished_containers_ms',
+                floatingLabelText: 'Interval for clearing finished containers (ms)',
+                info: 'Interval (in ms) to search and delete finished tests containers. Value of 0 means no auto clearing enabled',
+                valueType: 'int'
+            },
+            {
+                name: 'allow_insecure_tls',
+                key: 'allow_insecure_tls',
+                floatingLabelText: 'Allow insecure TLS',
+                info: 'If true, don\'t fail requests on unverified server certificate errors',
+                type: INPUT_TYPES.SWITCHER
+            },
+            {
+                category: 'Benchmark',
```

Comment:
add link to `benchmark` topic in out documentation page https://zooz.github.io/predator/benchmarks.html (Still need to create the page - but that should be the link)